crates/ln-dlc-node/src/ln/coordinator_event_handler.rs

```diff
@@ -345,14 +355,31 @@ fn handle_open_channel_request<S: TenTenOneStorage, N: Storage>(
     funding_satoshis: u64,
     push_msat: u64,
     temporary_channel_id: [u8; 32],
+    channel_type: ChannelTypeFeatures,
 ) -> Result<()> {
     let counterparty = counterparty_node_id.to_string();
     tracing::info!(
         counterparty,
         funding_satoshis,
         push_msat,
-        "Accepting open channel request"
+        "Handling open channel request"
     );
+
+    if channel_type.requires_zero_conf() {
+        bail!("Rejecting open channel request. Reason: The coordinator does not accept zero conf inbound channels");
```

Comment:
🔧 It's weird to talk about "the coordinator" when this will be printed by the coordinator.
```suggestion
        bail!("Rejecting open channel request. Reason: we do not accept zero-conf inbound channels");
```